modules/module2/module2-08-true_or_false_facets.qmd

```diff
@@ -0,0 +1,191 @@
+---
+format:
+  live-html
+resources:
+  - ../utils.py
+  - ../data
+---
+
+<script src='../../src/quiz.js'></script>  
+
+# 4.1. Exercises
+
+## True or False: Facets
+
+<div id='mcq1'></div> 
+<script>
+    generateQuiz(
+        'mcq1', 
+        'True or False',  
+        'If you faceting and assign a colour channel to the same column, you will obtain subplots all in different colours.',  
+        { 
+        'True': '',
+        'False': 'Setting a colour channel to the same column as the facet will change each of the faceted plots to different colours.'
```

Comment:
```suggestion
        'False': 'Since there is only one group (based on the chosen column) in each faceted chart, there will also only be one color in each of the charts.'
```